modulemd/v2/include/private/modulemd-component-private.h

```diff
@@ -119,3 +119,15 @@ gboolean
 modulemd_component_emit_yaml_build_common (ModulemdComponent *self,
                                            yaml_emitter_t *emitter,
                                            GError **error);
+
+/**
+ * modulemd_component_equals_wrapper:
+ * @a: A void pointer
+ * @b: A void pointer
+ *
+ * Returns: TRUE, if both the pointers are equal. FALSE, otherwise
+ *
+ * Since: 2.3
```

Comment:
This is correctly private API, but note that `Since: 2.3` will no longer be accurate..
```suggestion
 * Since: 2.5
```